conda/shell/etc/fish/conf.d/conda.fish

```diff
@@ -87,11 +87,13 @@ end
 
 # Faster but less tested (?)
 function __fish_conda_commands
-  command echo -e "activate\ndeactivate" ;and ls --color=none (conda info --root)/bin/conda-* | sed -r 's/^.*conda-([a-z]+)/\1/'
+  string replace -r '.*_([a-z]+)\.py$' '$1' $_CONDA_ROOT/lib/python*/site-packages/conda/cli/main_*.py
+  echo activate
+  echo deactivate
```

Comment:
Maybe should also add here

    string replace -r '.*conda-([a-z]+)$' '$1' $_CONDA_ROOT/bin/conda-*

or something like that.  All of the commands that come with conda-build, for example, have an executable associated with them.  `bin/conda-build` and `bin/conda-skeleton`, for example.

Reply:
That's what we were using in the *slow* implementation, the `conda-*` binaries. I guess we have no reason to stop doing that.